hicommonwealth/commonwealth
# magic wallet prompt only shows when using magic

## Link to Issue
Closes: #10374 

## Description of Changes
- Added logic to only show Magic wallet propmpts when the current address being used is a magic wallet

## Test Plan
- Log in with a user that has a magic wallet and any other wallet connected to that account
- click into a community while using a Magic address and confirm you see the Open Wallet prompt in the upper right next to the address
- switch to a non-magic address and confirm you no longer see icon/prompt
- do the same with the user menu dropdown
- if not in a community with stake go to one
- open Buy stake modal with a Magic address
- confirm you see "Add Funds" text
- switch to non-magic address and confirm you no longer see the "Add Funds" text/button in the modal
